scripts/msg_status.py

```diff
@@ -19,7 +19,9 @@
     are locked, it gives the option to remove the lock.
     @author: Will Rogers
 '''
+from __future__ import print_function
 
+from builtins import next
```

Comment:
Where there's import changes, then we should try and follow [PEP8 ordering](https://peps.python.org/pep-0008/#imports), at least for the grouping. So `builtins` should go after `os` (standard lib) and just before `dirq` (3rd-party). There will be other places to pay attention to this.

Reply:
According to https://python-future.org/imports.html and https://python-future.org/overview.html. 

It was meant to be at the top of the file. Seems it is a Known pollution to make the code work in py2.

I would keep this in the same position and corrected the other position and differentiate the standard imports to the future relevant changes at the first.- Just to highlight in a31fe35e089d9960ae60d34a2107c44717747221

Reply:
Yeah, that's ok. As discussed, it's a bit like the `__future__` imports.